File: 백준/Bronze/1292. 쉽게 푸는 문제/쉽게 푸는 문제.py
import sys

input = lambda: sys.stdin.readline().strip()

lst = [0]

index = 0
for i in range(1, 1001):
    if index > 1000:
        break
    for j in range(i):
        lst.append(i)
        index += 1
        if index > 1000:
            break

a, b = map(int, input().split())

answer = 0
for i in range(a, b+1):
    answer += lst[i]

print(answer)

File: 백준/Bronze/1292. 쉽게 푸는 문제/쉽게 푸는 문제.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
	static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
	static StringTokenizer st;
	
	public static void main(String[] args) throws IOException {
		int[] lst = new int[1001];
		
		int index = 1;
		for (int i = 1; i < 1001; i++) {
			if (index > 1000) {
				break;
			}
			for (int j = 0; j < i; j++) {
				if (index > 1000) {
					break;
				}
				lst[index++] = i;
			}
		}
		
		st = new StringTokenizer(br.readLine());
		int A = Integer.parseInt(st.nextToken());
		int B = Integer.parseInt(st.nextToken());
		
		int answer = 0;
		for (int i = A; i <= B; i++) {
			answer += lst[i];
		}
		
		System.out.println(answer);
	} // end of main
} // end of class